staker/block_validator.go

```diff
@@ -934,8 +982,14 @@ validationsLoop:
 				validatorProfileRunningHist.Update(time.Now().UnixMilli() - startTsMilli)
 				nonBlockingTrigger(v.progressValidationsChan)
 			})
+			pos += 1
```

Comment:
Another important part is missing here: integrating https://github.com/OffchainLabs/nitro/compare/validator_buffering_fix
The idea there was to reduce the amount of memory used for validation requests after they are sent, which allows us to really increase the amount of "pending" validations. Without it memory usage could become the blocking factor.

Reply:
Done.